cdc/sinkv2/tablesink/progress_tracker.go

```diff
@@ -143,8 +143,12 @@ func (r *progressTracker) addResolvedTs(resolvedTs model.ResolvedTs) {
 	r.mu.Lock()
 	defer r.mu.Unlock()
 
-	if r.nextEventID == 0 {
-		r.lastMinResolvedTs = resolvedTs
+	// If there is no event or all events are flushed, we can update the resolved ts directly.
```

Comment:
It's incorrect. If `advance` is called next time, `lastMinResolvedTs` will regress.

Reply:
One line is mistake:
```go
        r.resolvedTsCache[tsCacheLen-1].resolvedTs = resolvedTs
```